flask_config.py

```diff
@@ -5,5 +5,8 @@
 class Config:
     """Base configuration variables."""
     SECRET_KEY = os.environ.get('SECRET_KEY')
+    boardId = os.environ.get('boardId')
+    key = os.environ.get('key')
+    token = os.environ.get('token')
     if not SECRET_KEY:
         raise ValueError("No SECRET_KEY set for Flask application. Did you forget to run setup.sh?")
```

Comment:
The integration test fails locally for me because this line is run when this file is first _imported_, which happens before the environment variables have been loaded as part of that test. If we're no longer using the `SECRET_KEY` we could get rid of that, but we'll also need to ensure the boardId etc. are set in a constructor for the class (the `__init__` method), and we'll probably also need to adjust how this class is loaded in

Reply:
This has been changed as part of this commit - f695a3f